frontend/lib/uast/chpl-syntax-printer.cpp

```diff
@@ -555,9 +555,14 @@ struct ChplSyntaxVisitor {
   void visit(const Class* node) {
     ss_ << "class ";
     ss_ << node->name() << " ";
-    if (node->parentClass() != nullptr) {
+    if (node->numInheritExprs() > 0) {
       ss_ << ": ";
-      printAst(node->parentClass());
+      bool printComma = false;
+
+      for (int i = 0; i < node->numInheritExprs(); i++) {
+        if (printComma) ss_ << ", ";
+        printAst(node->inheritExpr(i));
+      }
```

Comment:
You'll need similar support for Record now that they can inherit & could you add a TODO comment next to the Union code (unless you decide to do the uAST changes for unions now?)